torch/csrc/autograd/python_torch_functions_manual.cpp

```diff
@@ -692,7 +692,7 @@ static PyObject* THPVariable_numel(
   }
 
   if (r.idx == 0) {
-    return wrap(r.tensor(0).numel());
+    return py::cast(r.tensor(0).sym_numel()).release().ptr();
```

Comment:
Somehow wrap() was converting c10::SymInt into Tensor. 

We explicitly convert from c10::SymInt to python torch.SymInt.

Reply:
@ezyang, fix copied from THPVariable_numel() in python_variable_methods.cpp